src/QMCDrivers/QMCDriverInput.cpp

```diff
@@ -119,6 +120,14 @@ void QMCDriverInput::readXML(xmlNodePtr cur)
       {
         reset_random_ = true;
       }
+      // These complications are due to the need to support bare <esimator> nodes
+      else if (cname == "estimators" || cname == "estimator")
+      {
+	if (estimator_manager_input_)
+	  estimator_manager_input_->readXML(tcur);
+	else
+	  estimator_manager_input_ = std::optional<EstimatorManagerInput>(std::in_place, tcur);
```

Comment:
I'd like to keep standalone "estimator" without "estimators" out of the batched driver input style.

Reply:
I'd like to address that in a later PR because there will be a good number of system tests that should be updated.

Reply:
Also double check if there is any objections after https://github.com/QMCPACK/qmcpack/issues/3645#issuecomment-1129209017